README.md

```diff
@@ -273,6 +273,51 @@ make uninstall-mesh
 
 ## Contributing
 
+### Create a new version
```

Comment:
Would it be nice, to have a clear recommendation, doing that in sepeartate PRS?

E.g.:
* OLM (first, as you stated)
* image updates (each a small / focused PR for serving, client, eventing, kafka)? 
* Golang updates

Reply:
It already says:

> These steps can be done one after the other and do not have to be sent as one PR, to avoid clogging up the respective PR.